uint16 EYE_STATUS_NORMAL=0
uint16 EYE_STATUS_BLINK=1
uint16 EYE_STATUS_SURPRISED=2
uint16 EYE_STATUS_SLEEPY=3
uint16 EYE_STATUS_ANGRY=4
uint16 EYE_STATUS_SAD=5
uint16 EYE_STATUS_HAPPY=6
uint16 EYE_STATUS_TROUBLED=7
uint16 EYE_STATUS_DELIGHTED=8
uint16 EYE_STATUS_EXPECTING=9
uint16 EYE_STATUS_HEART=10
uint16 EYE_STATUS_SHINE1=11
uint16 EYE_STATUS_SHINE2=12
uint16 EYE_STATUS_FLUSTRATED=13
uint16 EYE_STATUS_BORING=14
uint16 EYE_STATUS_SERIOUS=15
uint16 EYE_STATUS_REGRET1=16
uint16 EYE_STATUS_REGRET2=17


